_posts/en/newsletters/2022-01-12-newsletter.md

```diff
@@ -0,0 +1,180 @@
+initialize the reconciliation state for them."
+  a2link="https://bitcoincore.reviews/23443#l-63"
+
+  q3="Why doesn't Erlay include a p2p protocol version bump?"
+  a3="A new protocol version is not necessary for things to work; nodes using
+Erlay would not incompatible with the existing protocol. Older nodes that
+don't understand Erlay messages such as `sendrecon` would simply ignore them
+and still be able to function normally."
+  a3link="https://bitcoincore.reviews/23443#l-78"
+
+  q4="What is the reason for generating local per-connection salts? How is it
+generated?"
+  a4="A connection's reconciliation salt is the combination of both peers'
+locally-generated salts and is used to create shortids for each transaction. The
+salted hash function used for shortids is designed to efficiently create compact
+ids, but not guaranteed to be secure against collisions if an attacker can
+control what the salt is. When both sides contribute to the salt, neither peer
+can control what the salt is. Locally, a new salt is generated for each
+connection so that the node cannot be fingerprinted this way."
+  a4link="https://bitcoincore.reviews/23443#l-93"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23882][] contains a curious update of the documentation
+  in validation.cpp regarding the operation of the testnet3.
+
+  Initially, it was possible for transactions to have identical content
+  and thus colliding txids. The duplication issue could especially occur
+  with coinbase transactions for which the composition of input and
+  outputs was completely up to the creator of a block template. The
+  blockchain contains two duplicate coinbase transactions, at height
+  91,842 and 91,880. They are identical to previous coinbase
+  transactions and overwrote existing coinbase outputs before they were
+  spent, reducing total available supply by ₿100.  These incidents
+  prompted the introduction of [BIP30][] which forbade duplicate
+  transactions which was implemented by checking for duplication, and
+  was subsequently effectively prevented by [BIP34][] which introduced
+  the requirement to feature the block's height as the first item in a
+  coinbase transaction's scriptSig, thus preventing duplication and
+  removing the need to perform checks for duplicates.
```

Comment:
This sentence is pretty hard to get through. Can we break it up?

Reply:
Yeah, I'll take another pass at it.